mithril-aggregator/src/dependency_injection/builder.rs

```diff
@@ -742,7 +742,7 @@ impl DependenciesBuilder {
 
     async fn build_transaction_parser(&mut self) -> Result<Arc<dyn TransactionParser>> {
         // TODO: 'allow_unparsable_block' parameter should be configurable and its default value set to false
```

Comment:
The comment should be updated since the default value is what's expected :).

Reply:
Done :heavy_check_mark: